quickstart/201-azfw-with-secure-hub/variables.tf

```diff
@@ -0,0 +1,30 @@
+// Create Variables for Location and Tags
+variable "location" {
+  default = "eastus"
+}
+variable "tags" {
+  default = {
+    environment = "dev"
+    costcenter  = "1234556677"
+    owner       = "cloud team"
+    workload    = "azure firewall"
+  }
+}
+
+// Create Firewall Variables
+variable "fw_sku" {
+  default = "Premium" # Valid values are Standard and Premium
+}
+
+// Create Virtual Machine Sku Size Variables
+variable "vm_size" {
+  default = "Standard_D2_v3"
+}
+
+// Create Admin Username and Password
+variable "admin_username" {
+  default = "azureuser"
+}
+variable "admin_password" {
```

Comment:
To maintain consistency with the sample showing how to create a Windows VM, please generate the password as in the following sample: https://learn.microsoft.com/en-us/azure/virtual-machines/windows/quick-create-terraform